src/PersonalAccessToken.php

```diff
@@ -61,9 +61,11 @@ public static function findToken($token)
 
         [$id, $token] = explode('|', $token, 2);
 
-        if ($instance = static::find($id)) {
+        if (is_numeric($id) && $instance = static::find($id)) {
```

Comment:
Since the primary key can also be a string, I think it will be better if we do it like this:

```
[$id, $token] = explode('|', $token, 2);

if(static::getKeyType() === 'int' && !is_int($id)) {
    return null;
}

if ($instance = static::find($id)) {
    return hash_equals($instance->token, hash('sha256', $token)) ? $instance : null;
}
```
@driesvints what do you think?

Reply:
You cannot call `getKeyType` statically